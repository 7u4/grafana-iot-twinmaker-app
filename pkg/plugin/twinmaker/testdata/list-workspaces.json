{
  "NextToken": null,
  "WorkspaceSummaries": [
    {
      "Arn": "arn:aws:iottwinmaker:us-east-1:166800769179:workspace/CookieFactory-11-16",
      "CreationDateTime": "2021-11-16T18:30:40.54Z",
      "Description": "Workspace to test TwinMaker preview",
      "UpdateDateTime": "2021-11-16T18:30:40.54Z",
      "WorkspaceId": "CookieFactory-11-16"
    }
  ]
}
