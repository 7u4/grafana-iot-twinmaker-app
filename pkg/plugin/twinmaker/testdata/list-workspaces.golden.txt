ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {
    "custom": {}
}
Name: 
Dimensions: 4 Fields by 1 Rows
+---------------------------------------------------------------------------+----------------------------------+-------------------------------------+---------------------+
| Name: arn                                                                 | Name: created                    | Name: description                   | Name: workspaceId   |
| Labels:                                                                   | Labels:                          | Labels:                             | Labels:             |
| Type: []*string                                                           | Type: []time.Time                | Type: []*string                     | Type: []*string     |
+---------------------------------------------------------------------------+----------------------------------+-------------------------------------+---------------------+
| arn:aws:iottwinmaker:us-east-1:166800769179:workspace/CookieFactory-11-16 | 2021-11-16 18:30:40.54 +0000 UTC | Workspace to test TwinMaker preview | CookieFactory-11-16 |
+---------------------------------------------------------------------------+----------------------------------+-------------------------------------+---------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////cAIAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAIAAAAADAAAATAAAACgAAAAEAAAAFP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAAA0/v//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAFT+//8IAAAAGAAAAA0AAAB7ImN1c3RvbSI6e319AAAABAAAAG1ldGEAAAAABAAAAFwBAADgAAAAaAAAAAQAAADG/v//FAAAAEAAAABAAAAAAAAFATwAAAABAAAABAAAALT+//8IAAAAFAAAAAsAAAB3b3Jrc3BhY2VJZAAEAAAAbmFtZQAAAAAAAAAAtP7//wsAAAB3b3Jrc3BhY2VJZAAm////FAAAAEAAAABAAAAAAAAFATwAAAABAAAABAAAABT///8IAAAAFAAAAAsAAABkZXNjcmlwdGlvbgAEAAAAbmFtZQAAAAAAAAAAFP///wsAAABkZXNjcmlwdGlvbgAAABIAGAAUAAAAEwAMAAAACAAEABIAAAAUAAAAPAAAAEQAAAAAAAAKRAAAAAEAAAAEAAAAiP///wgAAAAQAAAABwAAAGNyZWF0ZWQABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAwAHAAAAY3JlYXRlZAAAABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAAQAAAAEQAAAAAAAUBQAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAAAwAAAADAAAAYXJuAAQAAABuYW1lAAAAAAAAAAAEAAQABAAAAAMAAABhcm4AAAAAAP////9IAQAAFAAAAAAAAAAMABYAFAATAAwABAAMAAAAsAAAAAAAAAAUAAAAAAAAAwMACgAYAAwACAAEAAoAAAAUAAAAyAAAAAEAAAAAAAAAAAAAAAsAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIAAAAAAAAAAgAAAAAAAAAUAAAAAAAAABYAAAAAAAAAAAAAAAAAAAAWAAAAAAAAAAIAAAAAAAAAGAAAAAAAAAAAAAAAAAAAABgAAAAAAAAAAgAAAAAAAAAaAAAAAAAAAAoAAAAAAAAAJAAAAAAAAAAAAAAAAAAAACQAAAAAAAAAAgAAAAAAAAAmAAAAAAAAAAYAAAAAAAAAAAAAAAEAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAABJAAAAYXJuOmF3czppb3R0d2lubWFrZXI6dXMtZWFzdC0xOjE2NjgwMDc2OTE3OTp3b3Jrc3BhY2UvQ29va2llRmFjdG9yeS0xMS0xNgAAAAAAAAAA36N/Whq4FgAAAAAjAAAAV29ya3NwYWNlIHRvIHRlc3QgVHdpbk1ha2VyIHByZXZpZXcAAAAAAAAAAAATAAAAQ29va2llRmFjdG9yeS0xMS0xNgAAAAAAEAAAAAwAFAASAAwACAAEAAwAAAAQAAAALAAAADgAAAAAAAMAAQAAAIACAAAAAAAAUAEAAAAAAACwAAAAAAAAAAAAAAAAAAAAAAAKAAwAAAAIAAQACgAAAAgAAACAAAAAAwAAAEwAAAAoAAAABAAAABT+//8IAAAADAAAAAAAAAAAAAAABQAAAHJlZklkAAAANP7//wgAAAAMAAAAAAAAAAAAAAAEAAAAbmFtZQAAAABU/v//CAAAABgAAAANAAAAeyJjdXN0b20iOnt9fQAAAAQAAABtZXRhAAAAAAQAAABcAQAA4AAAAGgAAAAEAAAAxv7//xQAAABAAAAAQAAAAAAABQE8AAAAAQAAAAQAAAC0/v//CAAAABQAAAALAAAAd29ya3NwYWNlSWQABAAAAG5hbWUAAAAAAAAAALT+//8LAAAAd29ya3NwYWNlSWQAJv///xQAAABAAAAAQAAAAAAABQE8AAAAAQAAAAQAAAAU////CAAAABQAAAALAAAAZGVzY3JpcHRpb24ABAAAAG5hbWUAAAAAAAAAABT///8LAAAAZGVzY3JpcHRpb24AAAASABgAFAAAABMADAAAAAgABAASAAAAFAAAADwAAABEAAAAAAAACkQAAAABAAAABAAAAIj///8IAAAAEAAAAAcAAABjcmVhdGVkAAQAAABuYW1lAAAAAAAAAAAAAAYACAAGAAYAAAAAAAMABwAAAGNyZWF0ZWQAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAEAAAABEAAAAAAAFAUAAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAMAAAAAwAAAGFybgAEAAAAbmFtZQAAAAAAAAAABAAEAAQAAAADAAAAYXJuAJgCAABBUlJPVzE=
